src/util/versionstore.cpp

```diff
@@ -72,8 +72,12 @@ QString VersionStore::applicationName() {
 
 // static
 QString VersionStore::platform() {
-#ifdef __APPLE__
+#ifdef Q_OS_IOS
+    QString base = QStringLiteral("iOS");
+#elif defined(Q_OS_MACOS)
     QString base = QStringLiteral("macOS");
+#elif defined(__APPLE__)
```

Comment:
Is this APPLE branch to catch other Apple OSes like iPadOS?

Reply:
Yes, though iPadOS will fall under iOS from what I understand. Just felt weird to leave let that branch fall through, so why not handle it

Reply:
Than it's an unknown OS, which is already covered at the end.